lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -10,7 +10,7 @@ class DashboardGenerator < Rails::Generators::NamedBase
         enum: "Field::String",
         float: "Field::Number",
         integer: "Field::Number",
-        time: "Field::DateTime",
```

Comment:
So being able to decide if we want our time attribute to be of tye `time` or `datetime`?